mllib/src/main/scala/org/apache/spark/mllib/evaluation/BinaryClassificationMetrics.scala

```diff
@@ -53,6 +62,15 @@ class BinaryClassificationMetrics @Since("1.3.0") (
   @Since("1.0.0")
   def this(scoreAndLabels: RDD[(Double, Double)]) = this(scoreAndLabels, 0)
 
+  /**
+    * Retrieves the score and labels (for binary compatibility).
+    * @return The score and labels.
+    */
+  @Since("1.0.0")
```

Comment:
Ah, was this visible as a getter? OK this is probably fine but just wasn't clear if it even became visible; is it used outside the constructor?

Reply:
If I understand correctly, a getter is constructed in binary only if the field is used somewhere else outside the class, and apparently it was used as a getter.  However, I can't tell where it was used from the current code, perhaps it was used that way in the past in an earlier version of the code.

Reply:
Actually, if that was the case in binary, why wouldn't it complain about the code right now - hmm, not quite sure why MIMA needs this.

Reply:
What if you remove it? right now I don't see it used anywhere outside the class. However we do need to retain the constructor; maybe that was the MiMa issue?

Reply:
yes, MIMA was complaining, and when I added this the error was fixed

Reply:
Hm OK, I guess it was a getter after all. It looks like it's since 1.3.0 though.

Reply:
done, updated Since to 1.3.0